de.metas.fresh/de.metas.fresh.base/src/main/jasperreports/JSONWebServiceAdapter.xml

```diff
@@ -0,0 +1,2 @@
+<?xml version="1.0" encoding="UTF-8" ?>
+<jsonDataAdapter class="net.sf.jasperreports.data.json.JsonDataAdapterImpl"><name>JSONWebServiceAdapter</name><dataFile xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:type="httpDataLocation"><method>GET</method><url>http://localhost:8181/api/dataentry/byId/540573/2006775</url><username></username><password></password><header><name>Authorization</name><value>6ccc55f8ceb94ace92b42fea773b0be9</value></header></dataFile><language>json</language><useConnection>true</useConnection><timeZone xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xmlns:java="http://java.sun.com" xsi:type="java:java.lang.String">Europe/Athens</timeZone><locale xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xmlns:java="http://java.sun.com" xsi:type="java:java.lang.String">en_US</locale><selectExpression></selectExpression></jsonDataAdapter>
```

Comment:
why having that hardcoded URL?
`http://localhost:8181/api/dataentry/byId/540573/2006775`

Reply:
Was for testing on dev server; will not remain like that